edgexfoundry/app-rfid-llrp-inventory
# refactor!: Use latest SDK for flattened config stem

BREAKING CHANGE: location of service configuration in Consul changed
  Also required updating for v3 modules

Signed-off-by: Jim Wang <yutsung.jim.wang@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-rfid-llrp-inventory/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

- unit tests: run `make test`
- integration test:
  1. Run EdgeX non-secure stack so Consul is running
  1. Build the service locally `make build`
  1. Run `make run` and 
          verify that the service configuration is at http://localhost:8500/ui/dc1/kv/edgex/v3/


## New Dependency Instructions (If applicable)
